packages/amplify_api/ios/Classes/SwiftAmplifyApiPlugin.swift

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import Flutter
+import UIKit
+import Amplify
+import AmplifyPlugins
+
+public class SwiftAmplifyApiPlugin: NSObject, FlutterPlugin {
+    public static func register(with registrar: FlutterPluginRegistrar) {
+        let channel = FlutterMethodChannel(name: "com.amazonaws.amplify/api", binaryMessenger: registrar.messenger())
+        let instance = SwiftAmplifyApiPlugin()
+        registrar.addMethodCallDelegate(instance, channel: channel)
+        do {
+            try Amplify.add(plugin: AWSAPIPlugin())
+            print("Successfully added API Plugin")
+        } catch {
+            print("Failed to add Amplify API Plugin \(error)")
+        }
+    }
+    
+    public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
+        let arguments = call.arguments as! Dictionary<String,AnyObject>
+        switch call.method {
+        case "query":
+            query(flutterResult: result, request: arguments)
+        case "mutate":
+            mutate(flutterResult: result, request: arguments)
+        default:
+            result(FlutterMethodNotImplemented)
+        }
+    }
+    
+    func query(flutterResult: @escaping FlutterResult, request: [String: Any]) {
```

Comment:
Following the 'bridge' pattern we've been using may make unit testing these easier.

Reply:
Done